HHS/Head-Start-TTADP
# Add an Admin UI and API to test sending emails using AWS SES

## Description of change
Adds a section in the admin UI where sending emails using AWS SES can be tested. Since cloud.gov doesn't support adding  DKIM for its domain associated with non-production environments, we need to run a test in production. The goal is to deploy to production and test sending emails from the prod domain.

## How to test
- On Dev navigate to the Email section in the admin UI
- Fill in the form with a valid email address
- Note that there is an error
- Use the following address: 
success@simulator.amazonses.com
- Verify there is a success message or there is a successful entry in the log file
## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1924


## Checklists

### Every PR

- [n/a] Code is meaningfully tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
